### PR TITLE
docs(color): fix color rgb docs

### LINKED ISSUE #1957: docs are confusing about default parameters of faker.color.rgb
https://next.fakerjs.dev/api/color.html#rgb

options.prefix shows default `0x`

examples show both
```
faker.color.rgb() // => "#cdfcdc"
faker.color.rgb() // '0xffffFF'
```

In the code it actually seems to default to "#"

```
const {
      format = 'hex',
      includeAlpha = false,
      prefix = '#',
      casing = 'lower',
    } = options || {};
```

either
- change the docs to show "#" is default?
- change the code to show "0x" is default?

### PR DESCRIPTION
**Pull request by Shinigami92**
close #1957

### CODE REVIEW

File: src/modules/color/index.ts [hunk 1]
```diff
@@ -315,16 +315,16 @@ export class ColorModule {
    * Returns an RGB color.
    *
    * @param options Options object.
-   * @param options.prefix Prefix of the generated hex color. Only applied when `'hex'` format is used. Defaults to `'0x'`.
+   * @param options.prefix Prefix of the generated hex color. Only applied when `'hex'` format is used. Defaults to `'#'`.
    * @param options.casing Letter type case of the generated hex color. Only applied when `'hex'` format is used. Defaults to `'mixed'`.
```
Comment (matthewmayer) [inline src/modules/color/index.ts]: This also seems wrong. It defaults to lower

File: src/modules/color/index.ts [hunk 2]
```diff
@@ -240,16 +240,16 @@ export class ColorModule {
...
    * @param options.format Format of generated RGB color. Defaults to `hex`.
    * @param options.includeAlpha Adds an alpha value to the color (RGBA). Defaults to `false`.
    *
    * @example
-   * faker.color.rgb() // '0xffffFF'
-   * faker.color.rgb({ prefix: '#' }) // '#ffffFF'
-   * faker.color.rgb({ casing: 'upper' }) // '0xFFFFFF'
-   * faker.color.rgb({ casing: 'lower' }) // '0xffffff'
+   * faker.color.rgb() // '#ffffFF'
```
Comment (matthewmayer) [inline src/modules/color/index.ts]: Shall we make it not ffffff? Doesn't feel very random to always get white!

Also lowercase per other comment

**Review by matthewmayer — CHANGES_REQUESTED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

File: src/modules/color/index.ts [hunk 3]
```diff
@@ -240,34 +240,34 @@ export class ColorModule {
...
-   * faker.color.rgb({ format: 'css' }) // 'rgb(255, 0, 0)'
-   * faker.color.rgb({ format: 'binary' }) // '10000000 00000000 11111111'
+   * faker.color.rgb() // '#8be4ab'
+   * faker.color.rgb({ prefix: '0x' }) // '0x9ddc8b'
+   * faker.color.rgb({ casing: 'upper' }) // '#B8A51E'
+   * faker.color.rgb({ casing: 'lower' }) // '#b12f8b'
+   * faker.color.rgb({ prefix: '#', casing: 'lower' }) // '#eb0c16'
+   * faker.color.rgb({ format: 'hex', casing: 'lower' }) // '#bb9d17'
+   * faker.color.rgb({ format: 'css' }) // 'rgb(216, 17, 192)'
+   * faker.color.rgb({ format: 'binary' }) // '00110010 00001000 01110110'
```
Comment (matthewmayer) [inline src/modules/color/index.ts:256]: can we add some `includeAlpha` examples too?

File: src/modules/color/index.ts [hunk 4]
```diff
@@ -240,34 +240,34 @@ export class ColorModule {
    * Returns an RGB color.
    *
    * @param options Options object.
-   * @param options.prefix Prefix of the generated hex color. Only applied when 'hex' format is used. Defaults to `'0x'`.
-   * @param options.casing Letter type case of the generated hex color. Only applied when `'hex'` format is used. Defaults to `'mixed'`.
+   * @param options.prefix Prefix of the generated hex color. Only applied when 'hex' format is used. Defaults to `'#'`.
+   * @param options.casing Letter type case of the generated hex color. Only applied when `'hex'` format is used. Defaults to `'lower'`.
```
Comment (matthewmayer) [inline src/modules/color/index.ts:244]: can we explicitly list what the possible values of `casing` and `format` are? (this is similar to issue we had in #1732 where we said we'd list out values of types manually until we are able to get the automatic extraction working)

image

**Review by matthewmayer — DISMISSED** [review]

File: src/modules/color/index.ts (hunk 3 above)
Comment (Shinigami92) [reply src/modules/color/index.ts:256]: This PR just fix the current docs
If we want to add something to the docs additionally, I suggest opening a separate PR

File: src/modules/color/index.ts (hunk 4 above)
Comment (Shinigami92) [reply src/modules/color/index.ts:244]: Somehow `options.sex` for `fullName` can also be correctly extracted: https://next.fakerjs.dev/api/person.html#fullname

So maybe @ST-DDT as the author of the docs gen code need to handle this in a separate PR

Comment (ST-DDT) [reply src/modules/color/index.ts:244]: I'll have a look soon.

Comment (ST-DDT) [reply src/modules/color/index.ts:244]: I created a PR for that:

- https://github.com/faker-js/faker/pull/1965

But it isn't that simple, so I need some feedback.

File: src/modules/color/index.ts (hunk 3 above)
Comment (ST-DDT) [reply src/modules/color/index.ts:256]: Done: https://github.com/faker-js/faker/issues/1967

**Pull request merged**